app/components/provider_interface/application_status_tag_component.rb

```diff
@@ -16,7 +16,7 @@ def colour
         # will never be visible to the provider
       when 'awaiting_provider_decision'
         'purple'
-      when 'interviewing', 'offer_deferred'
+      when 'interviewing', 'offer_deferred', 'inactive'
```

Comment:
This one and in other interfaces are just a placeholder. Because we're not setting the inactive status but the code magically picks up the enums I choose a random colour. This will be implemented when we show the inactive status in each interface (there are cards for this).